Funcky.Test/Extensions/EnumerableExtensions/GetNonEnumeratedCountOrNoneTest.cs

```diff
@@ -0,0 +1,35 @@
+#if TRY_GET_NON_ENUMERATED_COUNT
+using FsCheck;
+using FsCheck.Xunit;
+using Funcky.Test.TestUtils;
+
+namespace Funcky.Test.Extensions.EnumerableExtensions;
+public class GetNonEnumeratedCountOrNoneTest
```

Comment:
Nitpick: missing newline

Reply:
Fixed in 88946dc75d4128c37a86c28aab005c4e9a203c44